examples/next/faust-nx/pages/sample-page.tsx

```diff
@@ -0,0 +1,30 @@
+import { GetStaticPropsContext } from 'next';
+import { getNextStaticProps } from 'faust-nx';
+import { useQuery, gql } from '@apollo/client';
+import client from 'client';
+
+export default function Page(props: any) {
+  const {data} = useQuery(Page.query);
+  const title = data?.page?.title;
+  const content = data?.page?.content;
+  return <div>
+    <h1>{title}</h1>
+    <div dangerouslySetInnerHTML={{ __html: content}} />
+  </div>
+}
+
+Page.query = gql`
```

Comment:
This `Page.query` and `Page.variables` pattern works well. We should incorporate this into our WP templates as well so that it is all consistent.

Reply:
Let's use the https://wpengine.atlassian.net/browse/MERL-379 ticket to make it happen

Reply:
https://wpengine.atlassian.net/browse/MERL-436